src/Compilers/CSharp/Portable/Symbols/Source/TypeParameterConstraintClause.cs

```diff
@@ -55,6 +56,33 @@ internal enum TypeParameterConstraintKind
         AllNonNullableKinds = ReferenceType | ValueType | Constructor | Unmanaged,
     }
 
+    internal sealed class TypeParameterConstraintClauses
```

Comment:
>TypeParameterConstraintClauses [](start = 26, length = 30)

Would we need this type if we never cached "light-bound" constraints on a symbol? I would prefer us to never add it if we could do without.